### ISSUE TITLE
Unexpected token import

### ISSUE THREAD

**Issue by BorjaRafols** [xaksis/vue-good-wizard#5]
I'm getting this error using this package.

error during render : /login
/src/app/node_modules/vue-good-wizard/index.js:1
(function (exports, require, module, __filename, __dirname) { import GoodWizard from './components/Wizard.vue';
                                                              ^^^^^^

```
SyntaxError: Unexpected token import
    at createScript (vm.js:80:10)
    at Object.runInThisContext (vm.js:152:10)
    at Module._compile (module.js:605:28)
    at Object.Module._extensions..js (module.js:652:10)
    at Module.load (module.js:560:32)
    at tryModuleLoad (module.js:503:12)
    at Function.Module._load (module.js:495:3)
    at Module.require (module.js:585:17)
    at require (internal/module.js:11:18)
    at r (/src/app/node_modules/vue-server-renderer/build.js:8147:16)
    at Object.<anonymous> (webpack:/external "vue-good-wizard":1:0)
    at __webpack_require__ (webpack:/webpack/bootstrap 6c96c21018c6d7f6db3e:25:0)
    at Object.<anonymous> (src/app.js:4:0)
    at __webpack_require__ (webpack:/webpack/bootstrap 6c96c21018c6d7f6db3e:25:0)
    at Object.module.exports.Object.defineProperty.value (src/entry-server.js:1:0)
    at __webpack_require__ (webpack:/webpack/bootstrap 6c96c21018c6d7f6db3e:25:0)
    at module.exports.module.exports.list (webpack:/webpack/bootstrap 6c96c21018c6d7f6db3e:90:0)
    at Object.<anonymous> (server-bundle.js:95:10)
    at evaluateModule (/src/app/node_modules/vue-server-renderer/build.js:8155:21)
    at /src/app/node_modules/vue-server-renderer/build.js:8213:18
    at new Promise (<anonymous>)
    at /src/app/node_modules/vue-server-renderer/build.js:8205:14

```

So I guess this should be easily solvable by compiling your lovely component and adding the dist folder with ES5 syntax to the repo.

Could this be done?

**Comment by csgt** [comment]
I am having the same issue.  How did you manage to get it to work? Maybe we can do a PR.

**Comment by xaksis** [comment]
@BorjaRafols @csgt this should now be fixed in v1.2.0. the new version exposes the ES5 bundle

**Comment by BorjaRafols** [comment]
You rock! That was quick
